sources/AppBundle/Association/Model/User.php

```diff
@@ -514,7 +514,14 @@ public function getRoles()
             $defaultRoles[] = 'ROLE_ANTENNE';
         }
 
-        return array_unique(array_merge($this->roles, $defaultRoles));
+        $userRoles = array_filter(
```

Comment:
`$userRoles = array_diff($this->roles, ['ROLE_MEMBER_EXPIRED']);` serait sûrement + efficace, sauf si j'ai loupé autre chose ?